src/monopole_gr/monopole_gr.hpp

```diff
@@ -78,6 +78,25 @@
   beta^r = -(1/2) alpha r K^r_r
 
 
+  USING DENSITIZED CONSERVED VARS
+  --------------------------------
+
+  If we define:
+
+    rhob = sqrt(gamma) (tau + D)
+    Srb  = sqrt(gamma) S_r
+
+  then we can rewrite the eqns for a and K as
+
+  da/dr = a (4 - 4 a^2 + 3 a^2 (r K^r_r)^2 + 32 pi rhob)/(8 r)
+
+  d K^r_r = (8 pi Srb/a - 3 r K^r_r)/(r^2)
+
+  and the equation for alpha as
+
+  a^2 r alpha ((3/2) (K^r_r)^2 + 4 pi ((rho/(a r^2) + S)) +
+  ((1/a)(da/dr)r-2)dalpha/dr
+
```

Comment:
I did not check these equations.

Reply:
That's fine. They are unused for the moment. I just wanted it documented somewhere if/when we revisit. It's also in the Phoebus doc.